EEG101/src/redux/reducer.js

```diff
@@ -15,7 +15,9 @@ import {
   UPDATE_CLASSIFIER_DATA,
   SET_NATIVE_EMITTER,
   START_BCI_RUNNING,
-  STOP_BCI_RUNNING
+  STOP_BCI_RUNNING,
+  SET_REFRESH,
```

Comment:
I'd like to find a way to take refresh and noMore out of the Redux store. Because battery level only lives in the NavBar component, I think we should keep all state necessary for that functionality in the NavBar component as well.

Either that, or we move the battery level and battery listener outside of NavBar and into Redux

Reply:
Don't know redux that well unfortunately. Someone else would have to do that if it's possible and necessary.